index.html

```diff
@@ -301,17 +330,19 @@ <h2 class="top-heading">About &nbsp; &nbsp; &nbsp; &nbsp; Myself</h2>
               <span>Languages</span>
               <img class="skill-img1" src="./images/Union.png" alt="arrow" />
             </div>
-            <div class="javascript1">
-              <img src="./images/Ellipse 1.png" alt="js" />
-              <p>JavaScript</p>
-            </div>
-            <div class="html1">
-              <img src="./images/Ellipse 1 (1).png" alt="html" />
-              <p>HTML</p>
-            </div>
-            <div class="css1">
-              <img src="./images/Ellipse 1 (2).png" alt="css" />
-              <p>CSS</p>
+            <div class="Languages-desktop">
+              <div class="javascript1">
+                <img src="./images/Ellipse 1.png" alt="js" />
+                <p>JavaScript</p>
+              </div>
+              <div class="html1">
+                <img src="./images/Ellipse 1 (1).png" alt="html" />
+                <p>HTML</p>
+              </div>
+              <div class="css1">
```

Comment:
[Optional] Instead of creating multiple classes you can use a single class for all of the similar elements.